### DISCUSSION TITLE
Access a wallet without the WebApp where I created it.

### DISCUSSION THREAD

**Discussion by ander-db** [Web3Auth/Web3Auth#1231] [Help]
Hello! I'm trying to undestand how this technology actually works with the [diagram ](https://web3auth.io/docs/overview/how-web3auth-works) showed on the documentation  The diagram shows that the address is generated with the idToken, which mean that an address generated on a webapps are different between each other (using the same login provider). This raises two questions for me:
- Can a user access his wallet without login with the webapp used for creating the account? (I suppose that not)
- If this is the case, if the webapp disappear for any reason, those accounts will be inaccessible??

**Comment by Oberhaim** [comment]
If you find a work around for this please let me know ! 
been struggling with this. It feels so wrong to change addresses everytime.

**Reply by ander-db** [reply]
Hey! Having this approach (different addresses for each application) has its advantages and disadvantages. I made users log in with torus when they clicked on the social login, instead of web3auth directly. That worked for me.
